app/models/host/base.rb

```diff
@@ -91,7 +91,7 @@ def initialize(*args)
         primary_interface_attrs.each do |attr|
           values_for_primary_interface[attr] = new_attrs.delete(attr) if new_attrs.has_key?(attr)
         end
-        args.unshift(new_attrs.to_hash)
+        args.unshift(new_attrs.to_hash.with_indifferent_access)
```

Comment:
For the record, removed `to_hash` which was causing this. It is with indifferent access already (see above code).